.github/workflows/upload-aws-s3.yml

```diff
@@ -0,0 +1,18 @@
+name: "Upload builds"
+
+on: push
+
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: jakejarvis/s3-sync-action@master
+                with:
+                    args: --acl public-read --follow-symlinks --delete
+                env:
+                    AWS_S3_BUCKET: "z-lohika"
+                    DEST_DIR: "megazord/android/reloaded/${params.FLAVOR.toLowerCase()}/${params.BUILD_TYPE.toLowerCase()}/"
```

Comment:
As I understand from [here](https://github.com/wireapp/wire-android/blob/d4ec2b55a79258f20c2009be5e5419abb2ad3249/AR-builder.groovy#L391-L415) 

We should upload the APK to the path you use correctly in the PR, **plus** this other one as well (depending on [this](https://github.com/wireapp/wire-android/blob/d4ec2b55a79258f20c2009be5e5419abb2ad3249/AR-builder.groovy#L407) 

```
if (params.SOURCE_BRANCH.startsWith("PR-") || params.SOURCE_BRANCH == "develop" || params.SOURCE_BRANCH == "release/candidate") 
```

To (note the by-branch prefix)

path: "megazord/android/reloaded/**by-branch**/${params.SOURCE_BRANCH}/${params.FLAVOR.toLowerCase()}/${params.BUILD_TYPE.toLowerCase()}/"